source/framework/tools/src/TRestStringHelper.cxx

```diff
@@ -132,7 +132,7 @@ string REST_StringHelper::ReplaceMathematicalExpressions(string buffer, Int_t pr
         erased = true;
     }
 
-    for (int i = 0; i < Expressions.size(); i++) {
+    for (unsigned int i = 0; i < Expressions.size(); i++) {
```

Comment:
We should try to use `unsigned int` or `size_t` more consistently. My suggestion is to use `size_t` whenever we are dealing with the length of an array such as in this case and use `unsigned int` on the remaining cases.

```suggestion
    for (size_t i = 0; i < Expressions.size(); i++) {
```